packages/api/.eslintrc.yaml

```diff
@@ -0,0 +1,21 @@
+env:
```

Comment:
yaml plutôt que json ? on utilise json partout pour les configurations donc ce serait peut-être intéressant de garder le même format partout ?

Reply:
Ça m'irait de traduire en json. Je trouve yaml + lisible donc ça m'a aidé d'aller sur ce format, et ensuite c'était + fluide de rester dessus. Je ne mesure pas à quel point c'est embêtant d'avoir ce format. On l'avait déjà sur les github actions par ailleurs.
Bref, mon avis n'est pas très tranché ^^

Reply:
j'ai rien contre le yaml c'est juste que j'aime l'homogénéité. Que pense @VSerain sur le sujet ?